common/pc/laptop/default.nix

```
@@ -1,13 +1,6 @@
-{ config, lib, ... }:
-
+{ lib, config, ... }:
 {
   imports = [ ../. ];
 
-  # Gnome 40 introduced a new way of managing power, without tlp.
-  # However, these 2 services clash when enabled simultaneously.
-  # https://github.com/NixOS/nixos-hardware/issues/260
-  services.tlp.enable = lib.mkDefault (
-    (lib.versionOlder (lib.versions.majorMinor lib.version) "21.05")
-    || !config.services.power-profiles-daemon.enable
-  );
+  config.services.power-profiles-daemon.enable = lib.mkDefault (!config.services.tlp.enable);
```

What about conflicts with `services.auto-cpufreq.enable`, as already mentioned in https://github.com/NixOS/nixos-hardware/pull/1630#discussion_r2444855724?

Should `services.auto-cpufreq.enable` be ignored, blindly hoping `services.power-profiles-daemon.enable` and `services.auto-cpufreq.enable` work together, or should this conflict be explicitly handled with something like:

```suggestion
  config.services.power-profiles-daemon.enable = lib.mkDefault (
    !(config.services.auto-cpufreq.enable || config.services.tlp.enable)
  );
```